VTEX - Subscriptions API (v3).json

```diff
@@ -908,11 +939,30 @@
                 ],
                 "responses": {
                     "200": {
-                        "description": "Requested report",
+                        "description": "OK",
                         "content": {
                             "application/json": {
                                 "schema": {
                                     "$ref": "#/components/schemas/ReportResponse"
+                                },
+                                "example": {
+                                    "id": "7849df9f-00d4-11ef-8452-1254acca1d85",
+                                    "finished": false,
+                                    "recordsProcessed": 0,
+                                    "recordsSum": null,
+                                    "startDate": null,
+                                    "lastUpdateTime": null,
+                                    "completedDate": null,
+                                    "errorCount": null,
+                                    "percentageProcessed": 0,
+                                    "enqueueDate": "2024-04-22T18:16:50.7487312Z",
+                                    "email": "receiver@email.com",
+                                    "canceled": false,
+                                    "outputType": "CSV",
+                                    "zipped": false,
+                                    "linkToDownload": null,
+                                    "lastErrorMessage": null,
+                                    "statusMessage": "Report 7849df9f-00d4-11ef-8452-1254acca1d85 is being processed (0.00% completed). The file will be sent to receiver@email.com."
```

Comment:
substituir os `null`s por valores

Reply:
For this endpoint, The `null`s are the correct values, because the report has just been generated and there are no values applicable for them yet. The examples of other values instead of `null` can be found in the endpoint [Get report document details](https://developers.vtex.com/docs/api-reference/subscriptions-api-v3#get-/api/rns/pvt/reports/-reportName-/documents/-documentId-), when they get the document ID generated, consult it and the values have been generated.